legend-engine-xt-elasticsearch/legend-engine-xt-elasticsearch-V7-protocol/pom.xml

```diff
@@ -126,10 +126,6 @@
         </dependency>
 
         <!-- JACKSON -->
-        <dependency>
-            <groupId>org.finos.legend.engine</groupId>
```

Comment:
Why are we touching all these packages? `xt-elastic-*`, `xt-relationalStore-*`, `xt-text-protocol`, and `xt-sql-postgres-server`?

Reply:
This was because of the change to objectMapperFactory in shared-core but I reverted it